test/floatfuncs.jl

```diff
@@ -68,3 +68,5 @@ for elty in (Float32,Float64)
         @test round(elty2,A) == fill(round(elty2,x),(10,10,10))
     end
 end
+
+@test float(Char) == Any
```

Comment:
Well, the function is defined [here](https://codecov.io/github/kshyatt/julia/base/float.jl?ref=a92c7ff518408182d2bcdf3cbd0dbe3f2c1c1f40#l-125).